package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@redocly/cli": "^1.9.1"
```

Comment:
Can we fix the version to the exact 1.9.1 to avoid any unexpected issue?

Reply:
Good point ! Fixed